### PR TITLE
space age

### PR DESCRIPTION
**Pull request by qjd2413**

### CODE REVIEW

File: exercises/space-age/space_age_tests.plt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+:- begin_tests(space_age).
+:- use_module(library(plunit)).
+
+    floatEq(Real, Expected) :-
```
Comment (parkerl) [inline exercises/space-age/space_age_tests.plt]: Snake case to `float_eq` please?

**Comment by qjd2413** [comment]
I'm going to hold off on fixing the merge conflicts until other branches have been merged, to avoid spamming the log with `Merge branch 'master' into <branch>`

**Comment by parkerl** [comment]
@qjd2413 you can avoid the log spam with `git rebase master`. That is assuming you have the branch checked out. You will then probably have to force push the branch with `git push -f origin <branch name>`. 

Also, I have been squashing your commits into a single commit when merging so master does not have all the noise.

**Comment by parkerl** [comment]
:tada:

**Pull request merged**